src/dbnode/storage/index.go

```diff
@@ -1740,7 +1680,7 @@ func (i *nsIndex) execBlockQueryFn(
 
 func (i *nsIndex) execBlockWideQueryFn(
 	ctx context.Context,
-	cancellable *xresource.CancellableLifetime,
+	_ *xresource.CancellableLifetime,
```

Comment:
Should we just remove these from the exec block query fn's since they are unused now? Or is this just here until aggregate does not use cancellable anymore?